src/YiiElasticSearch/MultiSearchDataProvider.php

```diff
@@ -0,0 +1,133 @@
+<?php
+namespace YiiElasticSearch;
+
+/**
+ * A Data provider for searching into multiple indices
+ */
+class MultiSearchDataProvider extends DataProvider
+{
+    public $connID = 'elasticSearch';
```

Comment:
@tavantzo Hmm. Usually the connection Id is defined in the model. Can't you fetch it from the 1st model in `$models`?